fuse_examples/classification/knight/eval/eval.py

```diff
@@ -152,7 +152,7 @@ def decode_results(results: dict, output_dir: str, task1: bool, task2: bool) ->
 
     return results_table, results_text
 
-def eval(task1_prediction_filename: str, task2_prediction_filename: str, target_filename: str, output_dir: str, samples_descr_source: Union[str, List[str]] = "task1_pred") -> Tuple[OrderedDict, str]:
+def eval(task1_prediction_filename: str, task2_prediction_filename: str, target_filename: str, output_dir: str, samples_descr_source: Union[str, List[str]] = None) -> Tuple[OrderedDict, str]:
```

Comment:
Can we add None to sample_descr_source types and add the option in documentation.
And maybe change also "samples_descr" to "case_ids" in the comments (maybe also in the code?)
What do you think?

Reply:
done